app/src/main/java/ch/epfl/sdp/game/Server.java

```diff
@@ -85,6 +89,13 @@ private void initEnvironment() {
             } else Log.d(TAG, "initEnvironment: failed" + value.getException().getMessage()); });
     }
 
+    // unused for now
+    private void initMarket() {
+        // Market ------------------------------- NOTE LOCATION IMPORTANT FOR (android) MARKET ACTIVITY TEST
+        Market market = new Market(new GeoPoint(6.14, 46.22));
```

Comment:
Maybe add a TODO here as at some point, we will need to create a Market based on the position of the game area or something like this (if the map is not Geneva for exemple). Thus this test will fail.

Reply:
Normally, in the master branch, you should have a method called randomGeoPointAroundLocation() (inside the randomGenerator class). This method takes a GeoPoint and returns another random GeoPoint which is close the the one given as the input. Maybe you can use this and put the current position of the player as the input to that function.